src/Time/StepControllers/StepController.hpp

```diff
@@ -38,7 +41,8 @@ class StepController {
   StepController(StepController&&) = default;
   StepController& operator=(const StepController&) = default;
   StepController& operator=(StepController&&) = default;
-  virtual ~StepController() = default;
+  ~StepController() override = default;
```

Comment:
I guess we now have to decide on whether to have any of the constructors here since the item @kidder found suggests we don't mark the destructor override since that's implicit. I'm fine saying that we only mark the base class destructor as virtual and since in this case PUP::able is the base class, we don't do anything.